bin/cml.js

```diff
@@ -76,9 +76,43 @@ const setupLogger = (opts) => {
   });
 };
 
-const setupTelemetry = async (opts) => {
-  const { cml, cmlCommand: action } = opts;
-  opts.telemetryEvent = await jitsuEventPayload({ action, cml });
+const setupTelemetry = async (opts, { parsed: { defaulted } }) => {
```

Comment:
Why such a complex signature?

```js
{ parsed: { defaulted } 
```

do we expect `parsed` to be more complex?

Reply:
How would you write it? Just take `parsed` or the whole `yargs` and destructure `defaulted` in the function body?